model_compression_toolkit/keras/gradient_ptq/graph_info.py

```diff
@@ -75,7 +75,9 @@ def get_trainable_parameters(fxp_model: Model,
             # collect trainable weights per layer
             layer_trainable_weights = layer.quantize_config.get_trainable_quantizer_parameters()
             if add_bias:
-                use_bias = isinstance(layer.layer, tuple(fw_info.kernel_ops)) and layer.layer.get_config().get(USE_BIAS)
+                kernel_ops_attrs = fw_info.kernel_ops_attributes_mapping.get(type(layer.layer))
+                use_bias = kernel_ops_attrs is not None and kernel_ops_attrs[0] is not None \
+                           and layer.layer.get_config().get(USE_BIAS)
```

Comment:
I'm not sure why do we even check here if it's a "kernel op". @elad-c why is this necessary?
But if so, we can rewrite:

```
kernel_ops_attrs is not None and kernel_ops_attrs[0] is not None \
                           and layer.layer.get_config().get(USE_BIAS)
```

to:
```
type(layer.layer) in fw_info.kernel_ops_attributes_mapping and layer.layer.get_config().get(USE_BIAS)
```